infrastructure/scripts/check-docker-size.awk

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/awk -f
+{
+	ACTUAL = $1
+	ACTUAL_IN_MB = ACTUAL / 1024 / 1024
+	TARGET_IN_MB = $2
+	DISCREPANCY_PERCENT = $3
+	DISCREPANCY_LIMIT_UPPER_MB = TARGET_IN_MB * (1 + DISCREPANCY_PERCENT / 100)
+	DISCREPANCY_LIMIT_LOWER_MB = TARGET_IN_MB * (1 - DISCREPANCY_PERCENT / 100)
+	if(ACTUAL_IN_MB > DISCREPANCY_LIMIT_UPPER_MB)
+	{
+		printf "Image too big. Actual size is %.2f MB, but target is %.2f MB (%.2f MB with discrepancy).", ACTUAL_IN_MB, TARGET_IN_MB, DISCREPANCY_LIMIT_UPPER_MB
```

Comment:
.2f implies a precision that we don't have. I would either do .1f or .0f. And I'd remove the space before "MB".
```suggestion
		printf "Image too big. Actual size is %.0fMB, but target is %.0fMB (%.0fMB with discrepancy).", ACTUAL_IN_MB, TARGET_IN_MB, DISCREPANCY_LIMIT_UPPER_MB
```